Matrix crashes due to missing json-schema node dependency

### Bug Report

<!-- Fill in the relevant information below to help triage your issue. -->

|    Q        |   A
|------------ | ------
| Version(s)  | 1.8.0

#### Summary

It seems that the node dependency `@cfworker/json-schema` is not installed when matrix generation ramps up.

#### Current behavior

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/action/node_modules/@cfworker/json-schema/src/types' imported from /action/index.js
```

#### How to reproduce

Run matrix generation on any project on github which uses this action.

https://github.com/boesing/psalm-plugin-stringf/pull/8/checks?check_run_id=3309644575

#### Expected behavior

<!-- What was the expected (correct) behavior? -->
Matrix generation works as expected.

--- a/index.js
+++ b/index.js
@@ -5,7 +5,6 @@ import checkRequirements from './src/check-requirements.js';
 import configGatherer from './src/config.js';
 import createJobs from './src/create-jobs.js';
 import {Validator} from "@cfworker/json-schema";
-import {OutputUnit} from "@cfworker/json-schema/src/types";
 
 /**
  * Do early composer.json schema validation to avoid unnecessary ramp-ups of jobs which may fail

